src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+@mixin animation ($time) {
+  animation: rotate $time infinite linear;
+}
+
+@mixin arrowsPositionAndWidth () {
+  position: absolute;
+  width: 10px;
+}
+
+@mixin circle() {
+  height: 20px;
+  width: 20px;
+  border: 2px solid black;
+  background-color: white;
+  border-radius: 50%;
+}
+
+@mixin flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin transform-arrows() {
+  transform-origin: bottom center;
+}

```

Comment:
it seems to me u can rewrite it into a placeholder, cause these mixins don't accept any values

Reply:
> it seems to me u can rewrite it into a placeholder, cause these mixins don't accept any values

thx for advice 👍